test/src/travis-dep-builder.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+#
+
+prefix=$HOME/local
+cachedir=$HOME/local/.cache
+
+#
+# Ordered list of software to download and install into $prefix.
+#  NOTE: Code currently assumes .tar.gz suffix...
+#
+downloads="\
+https://www.open-mpi.org/software/ompi/v2.0/downloads/openmpi-2.0.2.tar.gz"
+
+declare -A extra_configure_opts=(\
+)
+
+declare -r prog=${0##*/}
+declare -r long_opts="prefix:,cachedir:,verbose,printenv"
+declare -r short_opts="vp:c:P"
+declare -r usage="\
+\n
+Usage: $prog [OPTIONS]\n\
+Download and install to a local prefix (default=$prefix) dependencies\n\
+for building flux-framework/flux-core\n\
+\n\
+Options:\n\
+ -v, --verbose           Be verbose.\n\
+ -P, --printenv          Print environment variables to stdout\n\
+ -c, --cachedir=DIR      Check for precompiled dependency cache in DIR\n\
+ -e, --max-cache-age=N   Expire cache in N days from creation\n\
+ -p, --prefix=DIR        Install software into prefix\n
+"
+
+die() { echo -e "$prog: $@"; exit 1; }
+say() { echo -e "$prog: $@"; }
+debug() { test "$verbose" = "t" && say "$@"; }
+
+GETOPTS=`/usr/bin/getopt -u -o $short_opts -l $long_opts -n $prog -- $@`
+if test $? != 0; then
+    die "$usage"
+fi
+
+eval set -- "$GETOPTS"
+while true; do
+    case "$1" in
+      -v|--verbose)          verbose=t;     shift   ;;
+      -c|--cachedir)         cachedir="$2"; shift 2 ;;
+      -e|--max-cache-age)    cacheage="$2"; shift 2 ;;
+      -p|--prefix)           prefix="$2";   shift 2 ;;
+      -P|--printenv)         print_env=1;   shift   ;;
+      --)                    shift ; break;         ;;
+      *)                     die "Invalid option '$1'\n$usage" ;;
+    esac
+done
+
+print_env () {
+    echo "export LD_LIBRARY_PATH=${prefix}/lib:$LD_LIBRARY_PATH"
+    echo "export CPPFLAGS=-I${prefix}/include"
+    echo "export LDFLAGS=-L${prefix}/lib"
+    echo "export PKG_CONFIG_PATH=${prefix}/lib/pkgconfig:${prefix}/share/pkgconfig"
```

Comment:
Once I used pkgconfig when LaunchMON had gcrypt integrated in it directly. But this isn't the case any longer. This can be dropped.